packages/elements/src/react/common/form/index.tsx

```diff
@@ -552,6 +556,11 @@ type FormInputProps =
 const Input = React.forwardRef<React.ElementRef<typeof RadixControl>, FormInputProps>(
   (props: FormInputProps, forwardedRef) => {
     const clerk = useClerk();
+    const field = useInput(props);
+
+    const hasPasskeyAutofillProp = Boolean(field.props.autoComplete?.includes('webauthn'));
+    const allowedTypeForPasskey = (['text', 'email'] as FormInputProps['type'][]).includes(field.props.type);
```

Comment:
Should we consider `tel` here for phone numbers?

Reply:
I think you are correct